cobalt/cobalt_paths.h

```diff
@@ -0,0 +1,27 @@
+// Copyright 2021 The Chromium Authors
```

Comment:
I'm not 100% on this, but I assume we'd want a Cobalt copyright on the forked code